pendulum/utils/_compat.py

```diff
@@ -6,8 +6,20 @@
 PY38 = sys.version_info[:2] >= (3, 8)
 
 if sys.version_info < (3, 9):
+    import importlib_resources as resources
+
     from backports import zoneinfo
 else:
     import zoneinfo
 
-__all__ = ["zoneinfo"]
+    from importlib import resources
+
+
+if sys.version_info < (3, 8):
+    from typing_extensions import Literal
+    from typing_extensions import SupportsIndex
```

Comment:
This makes `typing_extensions` a runtime dependency. Is that desired?

Reply:
Probably not :laughing:. I will need to rework this PR anyway since typing fails depending on the version of Python it's run on.